.github/workflows/test-phpunit.yml

```diff
@@ -129,9 +132,10 @@ jobs:
           php-version: ${{ matrix.php-versions }}
           tools: composer, pecl
           extensions: imagick, sqlsrv, gd, sqlite3, redis, memcached, oci8, pgsql
-          coverage: xdebug
+          coverage: ${{ env.COVERAGE_DRIVER }}
         env:
           update: true
+          COVERAGE_DRIVER: ${{ matrix.php-versions == env.COVERAGE_PHP_VERSION && 'xdebug' || 'none'}}
```

Comment:
Something seems odd here. Is this some special Actions ternary operator? It looks like a boolean doomed to fail.

Reply:
Environment variables here do not support ternary operators. This is the "recommended hack".